projects/ag-grid/documentation/code-examples/data-entry-grid/data-manager-added/data-manager-data-entry-grid-docs-demo.component.ts

```diff
@@ -214,13 +214,15 @@ export class SkyDataManagerDataEntryGridDemoComponent implements OnInit {
   public gridApi?: GridApi;
   public gridInitialized = false;
   public gridOptions!: GridOptions;
+  public noRowsTemplate;
 
   constructor(
     private agGridService: SkyAgGridService,
     private changeDetector: ChangeDetectorRef,
     private dataManagerService: SkyDataManagerService,
     private modalService: SkyModalService
   ) {
+    this.noRowsTemplate = `No results found`;
```

Comment:
```suggestion
    this.noRowsTemplate = `No results found.`;
```

Reply:
fixed in commit [56545e6 - "add periods."](https://github.com/blackbaud/skyux-ag-grid/pull/193/commits/56545e6d112a5dc513982f5560e10cdc4a119b42)